molgenis-core-ui/src/main/java/org/molgenis/core/ui/admin/permission/PermissionManagerController.java

```diff
@@ -101,12 +104,56 @@ public String init(Model model)
 
 	@PreAuthorize("hasAnyRole('ROLE_SU')")
 	@Transactional(readOnly = true)
-	@GetMapping("/entityclass/group/{groupId}")
+	@GetMapping("/repository/group/{groupId}")
 	@ResponseBody
-	public Permissions getGroupEntityClassPermissions(@PathVariable String groupId)
+	public PermissionTable getGroupRepositoryPermissions(@PathVariable String groupId)
 	{
 		Sid sid = getSidForGroupId(groupId);
-		return getEntityTypePermissions(sid);
+		return getRepositoryPermissions(sid);
+	}
+
+	public static class PermissionTable
+	{
+		private final List<PermissionRow> permissions;
+
+		public PermissionTable(List<PermissionRow> permissions)
+		{
+
+			this.permissions = permissions;
+		}
+
+		public List<PermissionRow> getPermissions()
+		{
+			return permissions;
+		}
+	}
+
+	public static class PermissionRow
+	{
+		private final Resource resource;
+		private final int mask;
+
+		public PermissionRow(Resource resource, int mask)
+		{
+			this.resource = resource;
+			this.mask = mask;
+		}
+
+		public Resource getResource()
+		{
+			return resource;
+		}
+
+		public int getMask()
+		{
+			return mask;
+		}
+	}
+
+	public static class Resource
+	{
+		public String id;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make id a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)